pca_ica/extract_cells.m

```diff
@@ -0,0 +1,353 @@
+function inv_quality = extract_cells(movie_source,pca_source,varargin)
+
+%   Cell extraction method that is based on an algorithm that essentially
+%   solves the sparse source separation problem to find cell filters.
+%   Traces are then found by using least squares.
+%
+%   Inputs:
+%
+%       movie_source: path to the movie matrix (height x width x frames)
+%
+%       pca_sources: path to the pca .mat file. It is assumed that there is
+%       a variable named pca_filters that has size [# of PCs x # of pixels]
+%       in the .mat file.
+%       
+%
+%   Outputs:
+%
+%       Script saves the traces and filters in a .mat file.
+%
+%       inv_quality: Inverse quality metric that indicates how good each
+%       found potential is as far as the extraction algorithm is concerned.
+%       It is a [1 x # of extracted cells] array whose entries are
+%       normalized to [(1/# of pixels),1] range. This array is also plotted
+%       on the screen at the end of execution.
+%
+%   Hakan Inan (Aug 2015)
+%
+
+% Some parameters
+mem_occup_scale_CPU = 0.5; % Occupy this x available memory on RAM
+mem_occup_scale_GPU = 0.7; % Occupy this x available memory on GPU
+corr_thresh = 0.1;
+filter_thresh = 0.25;
+max_num = 1000;
+do_baseline_fix = 0;
+
+if ~isempty(varargin)
+    for k = 1:length(varargin)
+        switch varargin{k}
+            case 'fix_baseline'
+                do_baseline_fix = 1;
+        end
+    end
+end
+
+fprintf('%s: Loading movie singular vectors...\n',datestr(now));
+
+load(pca_source,'pca_info','pca_filters');
```

Comment:
If you rename `pca_filters`, `pca_traces`, etc. as suggested in #109, then you could write here:
```
pca = load(pca_source);
is_trimmed = pca.info.trim.enabled;
h = pca.info.movie_height;
...
```
which is a bit cleaner, in my opinion.